src/org/thoughtcrime/securesms/WebxdcActivity.java

```diff
@@ -121,6 +122,7 @@ protected void onCreate(Bundle state, boolean ready) {
     webSettings.setJavaScriptEnabled(true);
     webSettings.setAllowFileAccess(false);
     webSettings.setBlockNetworkLoads(!internetAccess);
+    webView.setNetworkAvailable(internetAccess);
```

Comment:
Minor remark: [This is only a hint for the WebView](https://developer.android.com/reference/android/webkit/WebView#setNetworkAvailable(boolean)) (similar to our maybe_network()) and doesn't restrict anything.

Simon added it because it cannot hurt, while my personal opinion would be to leave it out, since it doesn't do anything but looks like it does.

Reply:
Just add a comment then?

Reply:
it sets `window.navigator.isOnline` - i think, it is good if that one is `false`, this may help using existing code. but yes, we should add a comment, i do a commit.

Reply:
i added a comment.